### ISSUE TITLE
How to print query size and response size?

### ISSUE THREAD

**Issue by liuyi1231** [google/hintless_pir#10]
I really don't know how to print query size and response size?
Are request.ByteSize() and response.ByteSize() correct options？

**Comment by b5li** [comment]
Yes, you can also use `request.ByteSizeLong()` and `response.ByteSizeLong()`.
